browsergym/experiments/src/browsergym/experiments/benchmark/base.py

```diff
@@ -215,6 +219,10 @@ def subset_from_regexp(self, column, regexp):
         name="webarena",
         high_level_action_set_args=DEFAULT_HIGHLEVEL_ACTION_SET_ARGS["webarena"],
         is_multi_tab=True,
+        full_reset_script="""\
+import browsergym.webarena.instance
```

Comment:
wtf!!!
why not just a function that calls it?

Reply:
done